docs/devops/github-actions-overview.md

```diff
@@ -0,0 +1,214 @@
+---
+title: GitHub Actions and .NET
+description: Learn what role GitHub Actions play in .NET application development.
+author: IEvangelist
+ms.author: dapine
+ms.date: 07/06/2021
+ms.topic: overview
+---
+
+# GitHub Actions and .NET
+
+In this overview, you'll learn what role [GitHub Actions](https://docs.github.com/actions) play in .NET application development. GitHub Actions allow your source code repositories to automate continuous integration (CI) and continuous delivery (CD). Beyond that, GitHub Actions expose more advanced scenarios &mdash; providing hooks for automation with code reviews, branch management, and issue triaging. With your .NET source code in GitHub you can leverage GitHub Actions to in many ways.
+
+## GitHub Actions
+
+GitHub Actions represent standalone commands, such as:
+
+- [actions/checkout](https://github.com/actions/checkout) - This action checks-out your repository under `$GITHUB_WORKSPACE`, so your workflow can access it.
+- [actions/setup-dotnet](https://github.com/actions/setup-dotnet) - This action sets up a .NET CLI environment for use in actions.
+- [dotnet/versionsweeper](https://github.com/dotnet/versionsweeper) - This action sweeps .NET repos for out-of-support target versions of .NET.
+
+> [!TIP]
+> The .NET docs team is utilizing the .NET version sweeper GitHub Action to automate issue creation. It runs on a schedule (as a CRON job). When it detects that .NET projects are targeting out-of-support versions &mdash; it creates issues reporting its findings. This is configurable, and helpful for tracking .NET version support concerns. It's available in the [GitHub Action Marketplace](https://github.com/marketplace/actions/net-version-sweeper).
```

Comment:
I don't think "cron" should be capitalized.  I'd check with @tdykstra to be sure.

Reply:
I would use lowercase -- cron.